service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/DocumentReferenceToNarrativeStatementMapper.java

```diff
@@ -47,7 +51,8 @@ public String mapDocumentReferenceToNarrativeStatement(final DocumentReference d
         final NarrativeStatementTemplateParametersBuilder builder = NarrativeStatementTemplateParameters.builder()
             .narrativeStatementId(narrativeStatementId)
             .availabilityTime(getAvailabilityTime(documentReference))
-            .hasReference(true);
+            .hasReference(true)
```

Comment:
Is it correct to set this to "true" if `buildParticipant(...)` can return an empty string [here](https://github.com/nhsconnect/integration-adaptor-gp2gp/pull/373/files#diff-aded27c693f2c511de5d4e319a801f6311120026c83bfb4b21a81bc5b620d77eR79) ?

Reply:
I believe so as reference is do to with the external document, not the participant. See this mustache template for narrative statement: https://github.com/nhsconnect/integration-adaptor-gp2gp/blob/main/service/src/main/resources/templates/ehr_narrative_statement_template.mustache